sending-a-simple-message-to-a-service/README.md

```diff
@@ -0,0 +1,325 @@
+# Sending a Simple Message to a Service
+
+## What you'll build
+
+Let's develop a service that allows a user to retrieve a list of doctors based on the doctor's specialization (category). The information about the doctors is retrieved from a separate microservice. 
```

Comment:
```suggestion
In this tutorial, you develop a service to retrieve a list of doctors based on the doctor's specialization (category). The information about the doctors is retrieved from a separate microservice. 
```
Again, it is about the tone used in technical docs. Also, it is a best practice to express the meaning in the minimum number of words possible avoiding redundant words.

Reply:
The suggestion and the original version don't mean the same thing IMO. We have to tall about the end-user experience.

Reply:
This is now

> In this tutorial, we will develop a service that allows a user to retrieve a list of doctors based on the doctor's specialization (category). The information about the doctors is retrieved from a separate microservice.

Reply:
We do not use the word `we` in technical docs. Should be active voice in `you` always.

Reply:
Changed to "you will develop".